store/map/index.js

```diff
@@ -132,14 +132,22 @@ export const actions = {
     })
   },
 
-  retrieveRasterLayerByImageId({ commit, state, getters }, { imageId, range }) {
+  retrieveRasterLayerByImageId({ commit, state, getters }, options) {
```

Comment:
Given the function name I would expect imageId as an argument. Maybe drop the ByImageId?

Reply:
Check